ToolTipMenu/ToolTipMenu.m

```diff
@@ -45,4 +45,12 @@ - (dispatch_queue_t)methodQueue
     [menuCont setMenuVisible:YES animated:YES];
 }
 
+RCT_EXPORT_METHOD(hide:(nonnull NSNumber *)reactTag){
+    
+    UIView *view = [self.bridge.uiManager viewForReactTag:reactTag];
+    [view becomeFirstResponder];
+    UIMenuController *menuCont = [UIMenuController sharedMenuController];
+    [menuCont setMenuVisible:NO animated:NO];
```

Comment:
do you need the view to becomeFirstResponder? just `setMenuVisible` should be enough? i'm not sure tho.